script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_1B.txt

```diff
@@ -145,7 +145,7 @@ Oh come on, this is a huge misunderstanding!
 Listen, I'm only going to say this once, I'm not a murderer or a crazy killer. You are――― you are the only person that I have ever killed in my life."
 <Page47>
 Arcueid's mouth hangs wide open.
-It seems that what I just said was very unexpected. The reality of that made her realize what she had done until now.
+It seems that what I just said was very unexpected. The reality of that made her realize what she had done until now. //Rephrase needed
```

Comment:
> It seems that what I just said has come as quite a shock. That revelation has forced her to finally realize what she's put me through up to this point.

Reply:
"That revelation" sounds like it's referring to the fact that "it seems that what I just said has come as quite a shock".
> It seems like I made her realized what she's put me through up to this point.